### ISSUE TITLE
Compilation Fails

### ISSUE THREAD

**Issue by psavva** [eu-digital-green-certificates/dgca-issuance-web#52]
## Describe the bug

I am unable to compile and serve the dgca issuance web application.

Error:

```
Compiled with warnings.

./node_modules/cbor/lib/constants.js
Module not found: Can't resolve 'bignumber.js' in 'C:\repos\eu-dgc\dgca-issuance-web\node_modules\cbor\lib'

src\api.tsx
  Line 22:8:  'axios' is defined but never used  @typescript-eslint/no-unused-vars

src\components\error-page.component.tsx
  Line 28:8:  'useNavigation' is defined but never used                                                                       @typescript-eslint/no-unused-vars
  Line 38:8:  React Hook React.useEffect has a missing dependency: 'props'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

src\components\notification-page.component.tsx
  Line 39:8:  React Hook React.useEffect has a missing dependency: 'props'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

src\components\record-recovery-cert-data.component.tsx
  Line 23:35:   'FormControlProps' is defined but never used                                                                                                                       @typescript-eslint/no-unused-vars
  Line 31:10:   'IdentifierType' is defined but never used                                                                                                                         @typescript-eslint/no-unused-vars
  Line 34:10:   'registerLocale' is defined but never used                                                                                                                         @typescript-eslint/no-unused-vars
  Line 38:32:   'DiseaseAgentTargeted' is defined but never used                                                                                                                   @typescript-eslint/no-unused-vars
  Line 111:8:   React Hook React.useEffect has a missing dependency: 'testCountryCode'. Either include it or remove the dependency array                                           react-hooks/exhaustive-deps     
  Line 118:8:   React Hook React.useEffect has missing dependencies: 'defaultTestCountryCode' and 'setDefaultTestCountryCode'. Either include them or remove the dependency array  react-hooks/exhaustive-deps     
  Line 157:11:  'handleError' is assigned a value but never used                                                                                                                   @typescript-eslint/no-unused-vars
  Line 260:11:  'formatDate' is assigned a value but never used                                                                                                                    @typescript-eslint/no-unused-vars

src\components\record-test-cert-data.component.tsx
  Line 23:35:   'FormControlProps' is defined but never used         @typescript-eslint/no-unused-vars
  Line 30:10:   'IdentifierType' is defined but never used           @typescript-eslint/no-unused-vars
  Line 33:10:   'registerLocale' is defined but never used           @typescript-eslint/no-unused-vars
  Line 39:17:   'VaccinationEntry' is defined but never used         @typescript-eslint/no-unused-vars
  Line 39:35:   'DiseaseAgentTargeted' is defined but never used     @typescript-eslint/no-unused-vars
  Line 88:12:   'vacLastDate' is assigned a value but never used     @typescript-eslint/no-unused-vars
  Line 88:25:   'setVacLastDate' is assigned a value but never used  @typescript-eslint/no-unused-vars
  Line 182:11:  'handleError' is assigned a value but never used     @typescript-eslint/no-unused-vars

src\components\record-vaccination-cert-data.component.tsx
  Line 23:35:   'FormControlProps' is defined but never used                                                                                                                           @typescript-eslint/no-unused-vars
  Line 31:10:   'IdentifierType' is defined but never used                                                                                                                             @typescript-eslint/no-unused-vars
  Line 34:10:   'registerLocale' is defined but never used                                                                                                                             @typescript-eslint/no-unused-vars
  Line 38:35:   'DiseaseAgentTargeted' is defined but never used                                                                                                                       @typescript-eslint/no-unused-vars
  Line 123:8:   React Hook React.useEffect has a missing dependency: 'issuerCountryCode'. Either include it or remove the dependency array                                             react-hooks/exhaustive-deps 
  Line 130:8:   React Hook React.useEffect has missing dependencies: 'defaultIssuerCountryCode' and 'setDefaultIssuerCountryCode'. Either include them or remove the dependency array  react-hooks/exhaustive-deps 
  Line 194:11:  'handleError' is assigned a value but never used                                                                                                                       @typescript-eslint/no-unused-vars
  Line 298:11:  'formatDate' is assigned a value but never used                                                                                                                        @typescript-eslint/no-unused-vars

src\components\show-certificate.component.tsx
  Line 59:12:  'dgci' is assigned a value but never used                                                                             @typescript-eslint/no-unused-vars
  Line 92:8:   React Hook React.useEffect has a missing dependency: 'handleError'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

src\index.tsx
  Line 24:25:  'HashRouter' is defined but never used  @typescript-eslint/no-unused-vars

src\misc\navigation.tsx
  Line 60:21:  'setMandant' is assigned a value but never used                                                                                            @typescript-eslint/no-unused-vars
  Line 78:8:   React Hook React.useEffect has missing dependencies: 'calculatedRoutes' and 'mandant'. Either include them or remove the dependency array  react-hooks/exhaustive-deps
  Line 96:8:   React Hook React.useEffect has missing dependencies: 'history' and 'routes'. Either include them or remove the dependency array            react-hooks/exhaustive-deps

src\misc\utils.tsx
  Line 40:1:  Assign object to a variable before exporting as module default  import/no-anonymous-default-export

src\misc\wbase45.tsx
  Line 32:15:  'x' is already defined                                          @typescript-eslint/no-redeclare
  Line 38:11:  'x' is already defined                                          @typescript-eslint/no-redeclare
  Line 39:12:  'd' is already defined                                          @typescript-eslint/no-redeclare
  Line 39:15:  'c' is already defined                                          @typescript-eslint/no-redeclare
  Line 64:12:  'i' is already defined                                          @typescript-eslint/no-redeclare
  Line 64:19:  'length' is already defined                                     @typescript-eslint/no-redeclare
  Line 84:1:   Assign object to a variable before exporting as module default  import/no-anonymous-default-export

src\root.component.tsx
  Line 28:8:  'LoginInterceptor' is defined but never used                                                                                                     @typescript-eslint/no-unused-vars
  Line 59:6:  React Hook React.useEffect has missing dependencies: 'setStoredMandant' and 'storedMandant'. Either include them or remove the dependency array  react-hooks/exhaustive-deps

src\routing.component.tsx
  Line 32:8:  'Header' is defined but never used        @typescript-eslint/no-unused-vars
  Line 35:8:  'PrivateRoute' is defined but never used  @typescript-eslint/no-unused-vars

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.

```

## Expected behaviour

The react app should run with no errors

## Steps to reproduce the issue

1) Checkout https://github.com/eu-digital-green-certificates/dgca-issuance-web/tree/0.0.5
2) npm install
3) npm start

## Technical details

```
C:\Users\psavva>systeminfo

Host Name:                 CS11299
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.19042 N/A Build 19042
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          csystem
Registered Organization:   N/A
Product ID:                00330-81490-46633-AA440
Original Install Date:     12/16/2020, 10:27:36 AM
System Boot Time:          5/6/2021, 8:36:41 AM
System Manufacturer:       LENOVO
System Model:              81Y4
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 165 Stepping 2 GenuineIntel ~2592 Mhz
BIOS Version:              LENOVO EGCN33WW, 12/24/2020
Windows Directory:         C:\WINDOWS
System Directory:          C:\WINDOWS\system32
Boot Device:               \Device\HarddiskVolume1
System Locale:             en-us;English (United States)
Input Locale:              en-us;English (United States)
Time Zone:                 (UTC+02:00) Athens, Bucharest
Total Physical Memory:     32,636 MB
Available Physical Memory: 19,289 MB
Virtual Memory: Max Size:  37,500 MB
Virtual Memory: Available: 22,045 MB
Virtual Memory: In Use:    15,455 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    WORKGROUP
Logon Server:              \\CS11299
Hotfix(s):                 10 Hotfix(s) Installed.
                           [01]: KB4601554
                           [02]: KB4562830
                           [03]: KB4570334
                           [04]: KB4577266
                           [05]: KB4577586
                           [06]: KB4580325
                           [07]: KB4593175
                           [08]: KB4598481
                           [09]: KB5001391
                           [10]: KB5003156
Network Card(s):           8 NIC(s) Installed.
                           [01]: Intel(R) Wi-Fi 6 AX201 160MHz
                                 Connection Name: Wi-Fi
                                 Status:          Media disconnected
                           [02]: Realtek PCIe GbE Family Controller
                                 Connection Name: Ethernet
                                 DHCP Enabled:    Yes
                                 DHCP Server:     10.16.1.118
                                 IP address(es)
                                 [01]: 10.16.20.216
                                 [02]: fe80::79d2:d59f:5cac:1251
                           [03]: Bluetooth Device (Personal Area Network)
                                 Connection Name: Bluetooth Network Connection
                                 Status:          Media disconnected
                           [04]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (Default Switch)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 172.23.208.1
                                 [02]: fe80::4123:ca71:1113:8d1
                           [05]: TAP-Windows Adapter V9 for OpenVPN Connect
                                 Connection Name: Local Area Connection 2
                                 Status:          Media disconnected
                           [06]: Fortinet Virtual Ethernet Adapter (NDIS 6.30)
                                 Connection Name: Ethernet 3
                                 Status:          Media disconnected
                           [07]: Fortinet SSL VPN Virtual Ethernet Adapter
                                 Connection Name: Ethernet 4
                                 Status:          Hardware not present
                           [08]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (WSL)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 172.29.128.1
                                 [02]: fe80::74b3:872e:26aa:652b
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.

```
## Possible Fix

Provide compilation instructions, as I assume my steps above are wrong?

## Additional context

```
PS C:\repos\eu-dgc\dgca-issuance-web> npm -v
6.14.9
```

```
PS C:\repos\eu-dgc\dgca-issuance-web> npm version
{
  'dgca-issuance-web': '0.1.0',
  npm: '6.14.9',
  ares: '1.16.1',
  brotli: '1.0.9',
  cldr: '37.0',
  icu: '67.1',
  llhttp: '2.1.3',
  modules: '83',
  napi: '7',
  nghttp2: '1.41.0',
  node: '14.15.2',
  openssl: '1.1.1g',
  tz: '2020a',
  unicode: '13.0',
  uv: '1.40.0',
  v8: '8.4.371.19-node.17',
  zlib: '1.2.11'
}
```

**Comment by daniel-eder** [comment]
@psavva I'm not a developer in this project, but I could see this being an issue with npm not handling package-lock.json, some legacy versions struggle with it.  

Could you try to 
1. delete the `node_modules` folder 
2. delete `package-lock.json` 
3. run `npm install`

If that does not work we'll have to wait for a dev to provide information on the target node version and other environment requirements.

**Comment by ggrund-tsi** [comment]
@psavva please try to use yarn for managing your packages.

1. delete the node_modules folder
2. delete package-lock.json
3. run npm install --global yarn
4. run yarn
5. run yarn start

I hope this will help you.

**Comment by psavva** [comment]
Hi @ggrund-tsi 

This is the result.
I deleted both node_modules folder as well as the pacakge-lock.json

then followed steps 3,4,5: Output as follows:

```
PS C:\repos\eu-dgc\dgca-issuance-web> npm install --global yarn

> yarn@1.22.10 preinstall C:\Users\psavva\AppData\Roaming\npm\node_modules\yarn
> :; (node ./preinstall.js > /dev/null 2>&1 || true)

C:\Users\psavva\AppData\Roaming\npm\yarn -> C:\Users\psavva\AppData\Roaming\npm\node_modules\yarn\bin\yarn.js
C:\Users\psavva\AppData\Roaming\npm\yarnpkg -> C:\Users\psavva\AppData\Roaming\npm\node_modules\yarn\bin\yarn.js
+ yarn@1.22.10
added 1 package in 2.226s
PS C:\repos\eu-dgc\dgca-issuance-web> yarn
yarn install v1.22.10
info No lockfile found.
[1/4] Resolving packages...
warning @types/react-datepicker > popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
warning node-sass > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
warning node-sass > node-gyp > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
warning node-sass > request > har-validator@5.1.5: this library is no longer supported
warning react-bootstrap > @types/classnames@2.3.1: This is a stub types definition. classnames provides its own type definitions, so you do not need this installed.
warning react-datepicker > react-popper > popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
warning react-scripts > babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
warning react-scripts > webpack-dev-server > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
warning react-scripts > webpack-dev-server > chokidar > fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
warning react-scripts > resolve-url-loader > rework > css > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
warning react-scripts > resolve-url-loader > rework > css > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
warning react-scripts > resolve-url-loader > rework > css > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
warning react-scripts > workbox-webpack-plugin > workbox-build > @hapi/joi@15.1.1: Switch to 'npm install joi'
warning react-scripts > workbox-webpack-plugin > workbox-build > rollup-plugin-babel@4.4.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-babel.
warning react-scripts > webpack > watchpack > watchpack-chokidar2 > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
warning react-scripts > workbox-webpack-plugin > workbox-build > @hapi/joi > @hapi/address@2.1.4: Moved to 'npm install @sideway/address'
warning react-scripts > workbox-webpack-plugin > workbox-build > @hapi/joi > @hapi/bourne@1.3.2: This version has been deprecated and is no longer supported or maintained
warning react-scripts > workbox-webpack-plugin > workbox-build > @hapi/joi > @hapi/hoek@8.5.1: This version has been deprecated and is no longer supported or maintained
warning react-scripts > workbox-webpack-plugin > workbox-build > @hapi/joi > @hapi/topo@3.1.6: This version has been deprecated and is no longer supported or maintained
warning react-scripts > workbox-webpack-plugin > workbox-build > @hapi/joi > @hapi/topo > @hapi/hoek@8.5.1: This version has been deprecated and is no longer supported or maintained
warning react-scripts > jest > jest-cli > jest-config > jest-environment-jsdom > jsdom > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
warning react-scripts > workbox-webpack-plugin > workbox-build > strip-comments > babel-plugin-transform-object-rest-spread > babel-runtime > core-js@2.6.12: core-js@<3.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.
warning react-scripts > jest > jest-cli > jest-config > jest-environment-jsdom > jsdom > request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
[2/4] Fetching packages...
info fsevents@2.3.2: The platform "win32" is incompatible with this module.
info "fsevents@2.3.2" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.2.13: The platform "win32" is incompatible with this module.
info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning " > @testing-library/user-event@12.8.3" has unmet peer dependency "@testing-library/dom@>=7.21.4".
warning " > bootstrap@4.6.0" has unmet peer dependency "jquery@1.9.1 - 3".
warning " > bootstrap@4.6.0" has unmet peer dependency "popper.js@^1.16.1".
warning " > react-moment@1.1.1" has unmet peer dependency "prop-types@^15.7.0".
warning " > react-qr-reader@2.2.1" has incorrect peer dependency "react@~16".
warning " > react-qr-reader@2.2.1" has incorrect peer dependency "react-dom@~16".
[4/4] Building fresh packages...
success Saved lockfile.
Done in 226.41s.
PS C:\repos\eu-dgc\dgca-issuance-web> yarn start
yarn run v1.22.10
$ react-scripts start
i ｢wds｣: Project is running at http://172.17.16.1/
i ｢wds｣: webpack output is served from
i ｢wds｣: Content not from webpack is served from C:\repos\eu-dgc\dgca-issuance-web\public
i ｢wds｣: 404s will fallback to /
Starting the development server...
Compiled with warnings.

./node_modules/cbor/lib/constants.js
Module not found: Can't resolve 'bignumber.js' in 'C:\repos\eu-dgc\dgca-issuance-web\node_modules\cbor\lib'

src\api.tsx
  Line 22:8:  'axios' is defined but never used  @typescript-eslint/no-unused-vars

src\components\error-page.component.tsx
  Line 28:8:  'useNavigation' is defined but never used                                                                       @typescript-eslint/no-unused-vars
  Line 38:8:  React Hook React.useEffect has a missing dependency: 'props'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

src\components\notification-page.component.tsx
  Line 39:8:  React Hook React.useEffect has a missing dependency: 'props'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

src\components\record-recovery-cert-data.component.tsx
  Line 23:35:   'FormControlProps' is defined but never used                                                                                                                       @typescript-eslint/no-unused-vars
  Line 31:10:   'IdentifierType' is defined but never used                                                                                                                         @typescript-eslint/no-unused-vars
  Line 34:10:   'registerLocale' is defined but never used                                                                                                                         @typescript-eslint/no-unused-vars
  Line 38:32:   'DiseaseAgentTargeted' is defined but never used                                                                                                                   @typescript-eslint/no-unused-vars
  Line 111:8:   React Hook React.useEffect has a missing dependency: 'testCountryCode'. Either include it or remove the dependency array                                           react-hooks/exhaustive-deps
  Line 118:8:   React Hook React.useEffect has missing dependencies: 'defaultTestCountryCode' and 'setDefaultTestCountryCode'. Either include them or remove the dependency array  react-hooks/exhaustive-deps
  Line 157:11:  'handleError' is assigned a value but never used                                                                                                                   @typescript-eslint/no-unused-vars
  Line 260:11:  'formatDate' is assigned a value but never used                                                                                                                    @typescript-eslint/no-unused-vars

src\components\record-test-cert-data.component.tsx
  Line 23:35:   'FormControlProps' is defined but never used         @typescript-eslint/no-unused-vars
  Line 30:10:   'IdentifierType' is defined but never used           @typescript-eslint/no-unused-vars
  Line 33:10:   'registerLocale' is defined but never used           @typescript-eslint/no-unused-vars
  Line 39:17:   'VaccinationEntry' is defined but never used         @typescript-eslint/no-unused-vars
  Line 39:35:   'DiseaseAgentTargeted' is defined but never used     @typescript-eslint/no-unused-vars
  Line 88:12:   'vacLastDate' is assigned a value but never used     @typescript-eslint/no-unused-vars
  Line 88:25:   'setVacLastDate' is assigned a value but never used  @typescript-eslint/no-unused-vars
  Line 182:11:  'handleError' is assigned a value but never used     @typescript-eslint/no-unused-vars

src\components\record-vaccination-cert-data.component.tsx
  Line 23:35:   'FormControlProps' is defined but never used                                                                                                                           @typescript-eslint/no-unused-vars
  Line 31:10:   'IdentifierType' is defined but never used                                                                                                                             @typescript-eslint/no-unused-vars
  Line 34:10:   'registerLocale' is defined but never used                                                                                                                             @typescript-eslint/no-unused-vars
  Line 38:35:   'DiseaseAgentTargeted' is defined but never used                                                                                                                       @typescript-eslint/no-unused-vars
  Line 123:8:   React Hook React.useEffect has a missing dependency: 'issuerCountryCode'. Either include it or remove the dependency array                                             react-hooks/exhaustive-deps
  Line 130:8:   React Hook React.useEffect has missing dependencies: 'defaultIssuerCountryCode' and 'setDefaultIssuerCountryCode'. Either include them or remove the dependency array  react-hooks/exhaustive-deps
  Line 194:11:  'handleError' is assigned a value but never used                                                                                                                       @typescript-eslint/no-unused-vars
  Line 298:11:  'formatDate' is assigned a value but never used                                                                                                                        @typescript-eslint/no-unused-vars

src\components\show-certificate.component.tsx
  Line 59:12:  'dgci' is assigned a value but never used                                                                             @typescript-eslint/no-unused-vars
  Line 92:8:   React Hook React.useEffect has a missing dependency: 'handleError'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

src\index.tsx
  Line 24:25:  'HashRouter' is defined but never used  @typescript-eslint/no-unused-vars

src\misc\navigation.tsx
  Line 60:21:  'setMandant' is assigned a value but never used                                                                                            @typescript-eslint/no-unused-vars
  Line 78:8:   React Hook React.useEffect has missing dependencies: 'calculatedRoutes' and 'mandant'. Either include them or remove the dependency array  react-hooks/exhaustive-deps
  Line 96:8:   React Hook React.useEffect has missing dependencies: 'history' and 'routes'. Either include them or remove the dependency array            react-hooks/exhaustive-deps

src\misc\utils.tsx
  Line 40:1:  Assign object to a variable before exporting as module default  import/no-anonymous-default-export

src\misc\wbase45.tsx
  Line 32:15:  'x' is already defined                                          @typescript-eslint/no-redeclare
  Line 38:11:  'x' is already defined                                          @typescript-eslint/no-redeclare
  Line 39:12:  'd' is already defined                                          @typescript-eslint/no-redeclare
  Line 39:15:  'c' is already defined                                          @typescript-eslint/no-redeclare
  Line 64:12:  'i' is already defined                                          @typescript-eslint/no-redeclare
  Line 64:19:  'length' is already defined                                     @typescript-eslint/no-redeclare
  Line 84:1:   Assign object to a variable before exporting as module default  import/no-anonymous-default-export

src\root.component.tsx
  Line 28:8:  'LoginInterceptor' is defined but never used                                                                                                     @typescript-eslint/no-unused-vars
  Line 59:6:  React Hook React.useEffect has missing dependencies: 'setStoredMandant' and 'storedMandant'. Either include them or remove the dependency array  react-hooks/exhaustive-deps

src\routing.component.tsx
  Line 32:8:  'Header' is defined but never used        @typescript-eslint/no-unused-vars
  Line 35:8:  'PrivateRoute' is defined but never used  @typescript-eslint/no-unused-vars

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.

```

http://localhost:3000/ reveals an empty React App

However, http://localhost:3000/swagger reveals a UI...

Is this expected?

**Comment by ggrund-tsi** [comment]
I've cleaned up the code a bit, removed all the warnings and unused dependencies. (and add bignumber.js)
So you can also run the code with npm.

Now the UI should appear without any additional string after the first '/'. (this was an relict)

**Comment by psavva** [comment]
Hi @ggrund-tsi  Can you confirm the PR Number, or Commit against which branch?

I also see that @jurosens  released a version 0.3.1 whilst the previous was 0.0.5, I cannot see the fix there either.
I think perhaps the numbering was a mistake? or was this intentional?
I cannot see the fix there either in any case

Best Regards
Panayiotis Savva
Cyprus

**Comment by ggrund-tsi** [comment]
Hi @psavva.
Actual it is in main branch an will deployed in next release. (maybe soon/today)
Regards, Gordon
